client/app/bundles/HelloWorld/components/ResturantsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+
+export default class ResturantsList extends React.Component {
+    componentDidMount(){
+        console.log("mounted");
+        console.log(this.props);
+    };
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            resturants: JSON.stringify(this.props.resturants)
```

Comment:
Like in your comment about RestaurantItem, we don't really need the state. The component is using props directly, as it should.